### DISCUSSION TITLE
Docker image for 0.15.4 missing?

### DISCUSSION THREAD

**Discussion by spoofedpacket** [go-graphite/carbonapi#654] [General]
The docker image for 0.15.4 seems to be missing:

https://hub.docker.com/r/gographite/carbonapi/tags

```
$ docker pull gographite/carbonapi:v0.15.4
Error response from daemon: manifest for gographite/carbonapi:v0.15.4 not found: manifest unknown: manifest unknown
```

**Comment by deniszh** [comment]
Unfortunately, Docker disabled automatic builds for free repositories.  We need to implement upload using GHA.

**Reply by spoofedpacket** [reply]
Oh, thats a pity. No probs!
